src/test/java/uk/gov/hmcts/darts/common/util/CommonTestDataUtil.java

```diff
@@ -151,10 +152,17 @@ public HearingEntity createHearing(String caseNumber, LocalTime time) {
         hearing1.setHearingDate(LocalDate.of(2023, 6, 20));
         hearing1.setScheduledStartTime(time);
         hearing1.setId(102);
+        hearing1.setTranscriptions(createTranscription());
         hearing1.addJudges(createJudges(2));
         return hearing1;
     }
 
+    public List<TranscriptionEntity> createTranscription() {
+        TranscriptionEntity transcription = new TranscriptionEntity();
+        transcription.setCompany("Transcription company");
```

Comment:
What happens to the `trt_id` column here - do we not need to set it?
Wondering if it's modelled wrong and 0 is going into the database row

Reply:
it doesn't matter at this stage as nothing is using the ID. This test data is for unit tests that don't get saved to the DB. If they were getting saved to the DB, we wouldn't want to set it, as database would assign the ID.
I have assigned it one now anyway.

Reply:
I was meaning the trt_id - what we have called transcriptionTypeId in our entity
Seeing as you have added an insert sql script for them also?

Reply:
done